support-frontend/assets/components/page/page.jsx

```diff
@@ -16,19 +16,26 @@ type PropTypes = {|
   header: Node,
   footer: Node,
   children: Node,
-  classModifiers: Array<?string>
+  classModifiers: Array<?string>,
+  backgroundImageSrc: ?string,
 |};
 
 
 // ----- Component ----- //
 
 export default function Page(props: PropTypes) {
+  const backgroundImage = props.backgroundImageSrc ? (
```

Comment:
This will break @jranks123's serverside rendering trick, which relies on the background image being in the CSS. https://github.com/guardian/support-frontend/pull/1804